README.md

```diff
@@ -250,6 +250,7 @@ Key-value protocol was developed and is used to provide communication layer betw
       * ```__testcase_start``` - sent by DUT, test case start data.
       * ```__testcase_finish``` - sent by DUT, test case result.
       * ```__exit``` - sent by DUT, test suite execution finished.
+      * ```__exit_event_queue``` - sent by host test, indicating no more events expected.
```

Comment:
Above documentation of ```__notify_complete``` is wrong. It is actually sent by the host test. Otherwise it will be redundant with ```__testcase_finish```. Also it does not matter as its purpose is to stop htrun event loop.

```__exit_event_queue``` is same as ```__notify_complete```.